packages/hoppscotch-common/src/services/spotlight/searchers/environment.searcher.ts

```diff
@@ -289,6 +295,45 @@ export class SwitchEnvSpotlightSearcherService
     }
   )[0]
 
+  fetchTeamEnvironmentList(
+    teamID: string,
+    fn: (results: TeamEnvironment[]) => void
+  ) {
+    const results: TeamEnvironment[] = []
+
+    runGQLQuery({
+      query: GetTeamEnvironmentsDocument,
+      variables: {
+        teamID: teamID,
+      },
+    }).then(
+      (result) => {
+        if (E.isRight(result)) {
+          if (result.right.team !== undefined && result.right.team !== null) {
+            results.push(
+              ...result.right.team.teamEnvironments.map(
+                (x) =>
+                  <TeamEnvironment>{
+                    id: x.id,
+                    teamID: x.teamID,
+                    environment: {
+                      name: x.name,
+                      variables: JSON.parse(x.variables),
+                    },
+                  }
```

Comment:
We can use the object shorthand notation here and destructure the parameters in the callback.